### PR TITLE
Apply comments and merge both branches up to date

### PR DESCRIPTION
**Pull request by anpineda**
Apply comments and merge both branches up to date

### CODE REVIEW

File: README.md
Comment (luvalverde) [inline README.md]: Why are you deleting this file?

File: LICENSE
Comment (luvalverde) [inline LICENSE]: Why are you deleting this file?

File: git-pre-commit-format
Comment (luvalverde) [inline git-pre-commit-format]: Why are you deleting this file?

File: localMatchApp/headers/classevent.h
Comment (luvalverde) [inline localMatchApp/headers/classevent.h]: Avoid writting code all wrapped together.
Use blank lines between each variable.
Apply this comment in all your PR.

File: localMatchApp/headers/classevent.h
Comment (luvalverde) [inline localMatchApp/headers/classevent.h]: According with the coding standard, files defining a class must be named as the class (ClassEvent.h).
Apply this comment in all your PR.

File: localMatchApp/headers/classevent.h
Comment (luvalverde) [inline localMatchApp/headers/classevent.h]: Each function must has its own description.
Apply this comment in all your PR.

File: localMatchApp/main.cpp
Comment (luvalverde) [inline localMatchApp/main.cpp]: Improve your variable names (a, w)

File: localMatchApp/sources/classevent.cpp
Comment (luvalverde) [inline localMatchApp/sources/classevent.cpp]: .cpp files should not contain functions descriptions. It is only at .h files.
Apply this comment in all your PR.

File: localMatchApp/sources/functionsevent.cpp
Comment (luvalverde) [inline localMatchApp/sources/functionsevent.cpp]: Add a comment describing what this does

File: localMatchApp/sources/homewindow.cpp
Comment (luvalverde) [inline localMatchApp/sources/homewindow.cpp]: ... the first ...

**Comment by anpineda** [comment]
I already add the files that were delete

Sorry but not sorry
PD: XOXO

File: localMatchApp/sources/homewindow.cpp
Comment (luvalverde) [inline localMatchApp/sources/homewindow.cpp]: remove extra blank lines

File: localMatchApp/sources/homewindow.cpp
Comment (luvalverde) [inline localMatchApp/sources/homewindow.cpp]: use a variable for the user name, so we then only need to update this variable value with the DB info

File: localMatchApp/sources/homewindow.cpp
Comment (luvalverde) [inline localMatchApp/sources/homewindow.cpp]: Add comments

File: localMatchApp/sources/homewindow.cpp
Comment (luvalverde) [inline localMatchApp/sources/homewindow.cpp]: Add comments

File: localMatchApp/sources/homewindow.cpp
Comment (luvalverde) [inline localMatchApp/sources/homewindow.cpp]: Add comments

File: localMatchApp/sources/homewindow.cpp
Comment (luvalverde) [inline localMatchApp/sources/homewindow.cpp]: According with the coding standard, braces must be on separate lines.
Apply this in all your PR

File: localMatchApp/sources/mainwindow.cpp
Comment (luvalverde) [inline localMatchApp/sources/mainwindow.cpp:47]: Add comments

File: localMatchApp/sources/mainwindow.cpp
Comment (luvalverde) [inline localMatchApp/sources/mainwindow.cpp:63]: Add comments

File: localMatchApp/sources/signup.cpp
Comment (luvalverde) [inline localMatchApp/sources/signup.cpp:18]: Avoid writting your code all wrapped together without blank lines.
It is more difficult to read

File: localMatchApp/sources/signup.cpp
Comment (luvalverde) [inline localMatchApp/sources/signup.cpp:56]: Add comments

File: localMatchApp/headers/classevent.h
Comment (luvalverde) [inline localMatchApp/headers/classevent.h]: According with the coding standard, all classes must have:
- copy constructor
- copy assignment operator
- destructor

Apply this comment in all your PR

**Review by luvalverde — CHANGES_REQUESTED** [review]
I added some comments.
Please improve your commit messages.

**Comment by luvalverde** [comment]
> I already add the files that were delete
> 
> Sorry but not sorry
> PD: XOXO

But I am sorry for your English grammar @anpineda :)

File: localMatchApp/headers/ClassEvent.h
Comment (wfallascr) [inline localMatchApp/headers/ClassEvent.h]: Please, provide more meaningful comments. "Declaration of the variable that" is unnecessary.

File: localMatchApp/headers/FunctionsEvent.h
Comment (wfallascr) [inline localMatchApp/headers/FunctionsEvent.h]: Functions?

File: localMatchApp/headers/FunctionsEvent.h
Comment (wfallascr) [inline localMatchApp/headers/FunctionsEvent.h]: This name does not reflect the utility of the class. Please, think on a new name for this class.

File: localMatchApp/headers/homewindow.h
Comment (wfallascr) [inline localMatchApp/headers/homewindow.h:34]: Why private slots?

File: localMatchApp/headers/homewindow.h
Comment (wfallascr) [inline localMatchApp/headers/homewindow.h]: Why you use a raw pointer here?

File: localMatchApp/sources/homewindow.cpp
Comment (wfallascr) [inline localMatchApp/sources/homewindow.cpp]: What exactly does this function? Manage is a too broad word.

File: localMatchApp/sources/homewindow.cpp
Comment (wfallascr) [inline localMatchApp/sources/homewindow.cpp]: Where is the _delete_ partner of this _new_?

File: localMatchApp/sources/homewindow.cpp
Comment (wfallascr) [inline localMatchApp/sources/homewindow.cpp]: Indentation problem? @Robchaves Please check this

File: localMatchApp/sources/homewindow.cpp
Comment (wfallascr) [inline localMatchApp/sources/homewindow.cpp]: Ths logic is still too big. Try creating smaller functions.

**Review by wfallascr — CHANGES_REQUESTED** [review]

**Comment by wfallascr** [comment]
@anpineda You changed the name of some files (added .txt), please return them to their original name

File: localMatchApp/headers/homewindow.h
Comment (masolano) [reply localMatchApp/headers/homewindow.h:34]: Because they have to be of the slot type since the signals of the buttons need it to be able to give functionality to the button, if they are only private it does not work

File: localMatchApp/headers/homewindow.h
Comment (masolano) [reply localMatchApp/headers/homewindow.h]: That part the program creates it alone, it is automatic

**Comment by masolano** [comment]
PR back to you

File: localMatchApp/CMakeLists.txt
Comment (luvalverde) [inline localMatchApp/CMakeLists.txt]: Remove this line.
It already exists in master

File: localMatchApp/headers/ClassEvent.h
Comment (wfallascr) [inline localMatchApp/headers/ClassEvent.h]: This comment is unnecessary

File: localMatchApp/headers/ClassEvent.h
Comment (wfallascr) [inline localMatchApp/headers/ClassEvent.h]: This comment is unnecessary

File: localMatchApp/headers/ClassEvent.h
Comment (wfallascr) [inline localMatchApp/headers/ClassEvent.h]: Name of this class is not good. It is evident that this is a class, so you do not need to add the word "Class". Think of something a little bit more descriptive, SocialEvent, GeneralEvent, UserEvent, etc

File: localMatchApp/headers/ClassEvent.h
Comment (wfallascr) [inline localMatchApp/headers/ClassEvent.h]: This comment is out of date

File: localMatchApp/headers/ClassEvent.h
Comment (wfallascr) [inline localMatchApp/headers/ClassEvent.h]: Guard's name is out of date

File: localMatchApp/headers/ClassEvent.h
Comment (wfallascr) [inline localMatchApp/headers/ClassEvent.h]: This is clearly a function, you do not need to restate that in the comment. Same for all your comments

File: localMatchApp/sources/ClassEvent.cpp
Comment (wfallascr) [inline localMatchApp/sources/ClassEvent.cpp]: The variable name should not start with a capital letter. Event -> event

File: localMatchApp/sources/ClassEvent.cpp
Comment (wfallascr) [inline localMatchApp/sources/ClassEvent.cpp]: This is a trivial constructor. [Read](https://en.cppreference.com/w/cpp/language/copy_constructor)

**Review by wfallascr — CHANGES_REQUESTED** [review]

File: localMatchApp/headers/ClassEvent.h
Comment (luvalverde) [inline localMatchApp/headers/ClassEvent.h]: remove extra blank line

File: localMatchApp/headers/ClassEvent.h
Comment (luvalverde) [inline localMatchApp/headers/ClassEvent.h]: As I told you in comments before, each function must have its description.
Provide a description for the set and another one for the get

File: localMatchApp/headers/ClassEvent.h
Comment (luvalverde) [inline localMatchApp/headers/ClassEvent.h]: As @wfallascr has told you in other comments, it is clear that these are variables, therefore you don't need to specify that in your comments (This variable saves ...)

File: localMatchApp/headers/ClassEvent.h
Comment (luvalverde) [inline localMatchApp/headers/ClassEvent.h]: remove extra blank lines

File: localMatchApp/headers/ClassEvent.h
Comment (luvalverde) [inline localMatchApp/headers/ClassEvent.h]: Remove this TODO and change the name of the variable.
The TODO should be inside the code where we need to change the path of the image for the image provided by the DB

File: localMatchApp/headers/FunctionsDatabase.h
Comment (luvalverde) [inline localMatchApp/headers/FunctionsDatabase.h]: check all your extra blank lines

File: localMatchApp/headers/FunctionsDatabase.h
Comment (luvalverde) [inline localMatchApp/headers/FunctionsDatabase.h]: same comment I gave you before in your other class

File: localMatchApp/headers/homewindow.h
Comment (luvalverde) [inline localMatchApp/headers/homewindow.h:27]: check all your extra blank lines in your PR

**Review by luvalverde — CHANGES_REQUESTED** [review]
Please avoid creating PR this huge.
It contains a lot of files and modifications.
The bigger the PR, the harder to catch errors in the code.

And please stop creating a lot of commit messages.
You need to get familiar creating PRs with only the necessarry commits

**Comment by masolano** [comment]
PR back to you

File: localMatchApp/headers/GeneralEvent.h
Comment (luvalverde) [inline localMatchApp/headers/GeneralEvent.h]: Please remove all "The class of ..." from all your classes description.

File: localMatchApp/headers/GeneralEvent.h
Comment (luvalverde) [inline localMatchApp/headers/GeneralEvent.h]: Be careful with your blank lines. You always have some

File: localMatchApp/headers/homewindow.h
Comment (luvalverde) [inline localMatchApp/headers/homewindow.h:40]: add blank lines between functions

File: localMatchApp/headers/homewindow.h
Comment (luvalverde) [inline localMatchApp/headers/homewindow.h:52]: missing description

File: localMatchApp/headers/FunctionsDatabase.h
Comment (wfallascr) [inline localMatchApp/headers/FunctionsDatabase.h]: This name is redundant. A class is composed of functions and variables. Think of something different, like DataBaseManager, etc

File: localMatchApp/headers/mainwindow.h
Comment (luvalverde) [inline localMatchApp/headers/mainwindow.h:51]: remove blank lines

File: localMatchApp/headers/signup.h
Comment (luvalverde) [inline localMatchApp/headers/signup.h:42]: missing description

File: localMatchApp/headers/FunctionsDatabase.h
Comment (wfallascr) [inline localMatchApp/headers/FunctionsDatabase.h]: Out of date

File: localMatchApp/headers/GeneralEvent.h
Comment (wfallascr) [inline localMatchApp/headers/GeneralEvent.h]: Out of date

File: localMatchApp/headers/signup.h
Comment (wfallascr) [inline localMatchApp/headers/signup.h]: Indentation problem

File: localMatchApp/sources/homewindow.cpp
Comment (luvalverde) [inline localMatchApp/sources/homewindow.cpp]: Please merge the 3 IF statements into just one to enable/disable the buttons.
You just need to compare if there is just one event, or if we are in the first or last event.
So, with an IF, ELSE IF, ELSE statements you can do the same

File: localMatchApp/sources/homewindow.cpp
Comment (wfallascr) [inline localMatchApp/sources/homewindow.cpp]: No sentence begin with "is", you are missing the subject

**Review by luvalverde — CHANGES_REQUESTED** [review]

**Review by wfallascr — CHANGES_REQUESTED** [review]

**Comment by masolano** [comment]
PR back to you

**Review by luvalverde — APPROVED** [review]
Just change your commit message

**Comment by masolano** [comment]
Ready

**Comment by luvalverde** [comment]
Please keep the commit header description brief and you then can explain yourself in the commit body message @masolano

**Comment by masolano** [comment]
Ready

**Review by luvalverde — APPROVED** [review]
Let's wait for @wfallascr review/approval

**Review by wfallascr — APPROVED** [review]

**Pull request merged**
